x/application/client/cli/tx_stake_application.go

```diff
@@ -43,7 +50,10 @@ $ poktrolld --home=$(POKTROLLD_HOME) tx application stake-application 1000upokt
 				return err
 			}
 
-			serviceIds := strings.Split(serviceIdsString, ",")
+			serviceIds, err := config.ParseApplicationConfigs(configContent)
```

Comment:
Right now `ApplicationServiceConfig` is built under the hood in `NewMsgStakeApplication`. Wdyt if updating `ParseApplicationConfigs` and propagating that directly so `serviceIds` is encapsulated in the business logic?